vpc/sg.tf

```diff
@@ -0,0 +1,65 @@
+resource "aws_security_group" "sg" {
```

Comment:
But than you need to best remove some rules like NRPE and Puppet. Those are not always needed